os_config_policy/main.tf

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+# Sample OS policy based on example - https://cloud.google.com/compute/docs/os-configuration-management/working-with-os-policies#example-2
```

Comment:
This URL is different from what is mentioned in the PR title.

https://cloud.google.com/compute/docs/os-configuration-management/create-os-policy-assignment